app/views/docs/permissions.phtml

```diff
@@ -23,18 +23,18 @@
 <div class="notice calm margin-top-large margin-bottom-large">
     <h4>Note</h4>
     
-    <p>You will need an active session to handle database transactions for your collections.</p>
-    <p>If you have rules where you allow for wild cards or role:guest in a collection, you can create an Anonymous Session before you make the API call.</p>
+    <p>You will need an active session to perform database transactions for your collections.</p>
```

Comment:
Well, you don't need a session to read. Can we rename it from "database transaction" to "database changes"? Or directly specify you need a session to create, update or delete?